packages/charts/test/styling.test.js

```diff
@@ -76,16 +76,9 @@ describe('vaadin-chart styling', () => {
         data: [29.9, 71.5, 106.4, 129.2, 144.0, 176.0, 135.6, 148.5, 216.4, 194.1, 95.6, 54.4],
       });
 
-      // As there's no --vaadin-charts-color-1 this should pick default styling
-      configuration.addSeries({
-        type: 'column',
-        data: [19.9, 61.5, 96.4, 119.2, 134.0, 166.0, 125.6, 138.5, 206.4, 184.1, 85.6, 44.4],
-      });
-
       const rects = chart.$.chart.querySelectorAll('.highcharts-legend-item > rect');
-      expect(rects).to.have.lengthOf(2);
+      expect(rects).to.have.lengthOf(1);
       expect(getComputedStyle(rects[0]).fill).to.equal('rgb(0, 255, 0)');
-      expect(getComputedStyle(rects[1]).fill).to.equal('rgb(22, 118, 243)');
```

Comment:
This tested the default color which was hardcoded in old core styles, with base styles it returns `oklch()` which can be probably removed as it's not really relevant for the test (the assertion above is enough).